.github/VelogCloneCoding.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Velog clone coding</title>
+
+    <link rel="stylesheet" href = "VelogCloneCoding.css">
+
+    <!-- 구글 icon 가져옴 -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0" />
+    
+</head>
+    
+    
+<body>
+    
+        <header id = "header">
+            <div class = "firstrow-container">
+                <h2>velog</h2>
+                <button class = "button-searching">
+                    <i class="material-symbols-outlined">search</i>
+                </button>
+
+                <button class = "button-new-article">
+                    새 글 작성
+                </button>
+
+                <button class = "button-my-account">
+                    <i class = "material-symbols-outlined">account_circle</i>
+                </button>
+            
+            </div>
+        
+            
+
+            
+    
+            <div class="button-container">
+                <button class="button-trending">
+                    <i class="material-symbols-outlined">trending_up</i>트렌딩</button>
+
+
+                <button class="button-recent">
+                    <i class="material-symbols-outlined">schedule</i>최신</button>
+
+
+                
+            </div>
+
+            
+            
+        </header>
+
+        <article>
+            <ul>
+                <li><p class = "post"></p> <p class = "post"> <p class = "post"> <p class = "post"> <p class = "post"></li>
```

Comment:
오옹 이미지로 ul 태그 사용해서 넣으셨군요!! ul 태그 사용해서 카드들 배열하는 것도 굉장히 재밌는 접근인 거 같아요! 배워갑니당:) 다음번에는 이미지랑 텍스트 부분이랑 각각의 요소들이 분리돼있는 container를 만들어서 다양한 포스팅 내용을 동적으로 담을 수 있게 설계 해보셔도 좋을 것 같아요 bb

이건 다른 분 리뷰 남긴 건데 비슷한 부분이 있는 것 같아 같이 남겨둡니다!


포스팅 카드 하나하나는 그 구조가 똑같기 때문에 
```suggestion
       </ul>
       <div class="posting-card"> //가장 큰 상자가 되는 컨테이너
           <img src = "이미지 링크"/> //이미지 박스 (img태그는 이미지 밖에 안 올려집니다)
           <div class="text-box"> // 이미지 밑에 텍스트 박스
               <p>타이틀</p>
               <p>설명</p> 
           </div>
       </div>
```

이런 식으로 제일 큰 박스인 posting-card div 태그 사이에 img 태그로 감싸진 img 부분과 text-box div 태그로 감싸진 부분 이렇게 두 가지 영역으로 나뉘고, 다시 text-box div 태그 안에 타이틀과 설명이 들어가는 방식입니다! 이게 바로 우리가 흔히 들은(?) 부모 요소와 자식 요소의 개념입니다. 서로 친구 관계여서 동등한 depth에 있는 건지, 자식이라서 부모의 성질을 계승하게 할 건지 등등을 고려하시면 좋습니당. 
만약에 posting-card의 배경색을 바꾸면 그 아래에 있는 모든 컴포넌트들의 배경색이 바뀌겠지만, text-box의 배경색을 바꾸면 부모 요소인 posting-card나 친구 요소(?)인 img 태그는 영향을 받지 않겠죠?!

이렇게 하나의 posting-card라는 큰 박스를 만들어놓으면, 이제 각기 다른 내용물이 들어가더라도 크게는 posting-card라는 컴포넌트가 반복이 되는 거죠! (물론 배치는 css로 조절)